### PR TITLE
Homework2 tools

### PR DESCRIPTION
**Pull request by danielzwierz**

### CODE REVIEW

**Review by Marcinati — APPROVED** [review]

File: vimtutor [hunk 1]
```diff
@@ -0,0 +1,970 @@
...
+  1. Move the cursor to the line below marked --->.
+
+  2. To fix the errors, move the cursor until it is on top of the
+     character to be deleted.
+
+  3. Press the	x  key to delete the unwanted character.
+
+  4. Repeat steps 2 through 4 until the sentence is correct.
+
+---> The ccow jumpedd ovverr thhe mooon.
```
Comment (ziobron) [inline vimtutor]: not completed

File: vimtutor [hunk 2]
```diff
@@ -0,0 +1,970 @@
...
+
+  2. To make the first line the same as the second, move the cursor on top
+     of the character BEFORE which the text is to be inserted.
+
+  3. Press  i  and type in the necessary additions.
+
+  4. As each error is fixed press <ESC> to return to Normal mode.
+     Repeat steps 2 through 4 to correct the sentence.
+
+---> There is text misng this .
```
Comment (ziobron) [inline vimtutor]: not completed

File: vimtutor [hunk 3]
```diff
@@ -0,0 +1,970 @@
...
+
+  3. As the text has been appended press <ESC> to return to Normal mode.
+
+  4. Move the cursor to the second line marked ---> and repeat
+     steps 2 and 3 to correct this sentence.
+
+---> There is some text missing from th
+     There is some text missing from this line.
+---> There is also some text miss
+     There is also some text missing here.
```
Comment (ziobron) [inline vimtutor]: not completed

**Review by ziobron — CHANGES_REQUESTED** [review]
It looks that you have not completed vimtutor. I can't see changes in this file. Please deliver them :)

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**